sdk/data/azcosmos/cosmos_container.go

```diff
@@ -358,6 +359,226 @@ func (c *ContainerClient) ReadItem(
 	return newItemResponse(azResponse)
 }
 
+// PatchItem patches an item in a Cosmos container.
+// ctx - The context for the request.
+// partitionKey - The partition key for the item.
+// itemId - The id of the item to patch.
+// item - The content to be used to patch.
+// o - Options for the operation.
+func (c *ContainerClient) PatchItem(
+	ctx context.Context,
+	partitionKey PartitionKey,
+	itemId string,
+	item []byte,
```

Comment:
Yes, I was thinking of making something similar but instead decided to leave it to user side.
Will try making some contract structs like java does.

Reply:
While giving the user the freedom makes the SDK simpler, it also opens the door for someone spending hours debugging an issue because they simply had a typo on the JSON contract. This will also reduce the number of methods in the container to just 1.

Reply:
See newly created `sdk/data/azcosmos/cosmos_item_request_patch_options.go`